packages/location-field/src/index.tsx

```diff
@@ -735,16 +765,14 @@ const LocationField = ({
       <S.InputGroup>
         {/* location field icon -- also serves as dropdown anchor */}
         <S.Dropdown
+          input={<>{textControl}{clearButton}</>}
```

Comment:
While this PR is open, what would be a good way to add support for the input's `required` attribute? This is in anticipation of form validation. An `isRequired` prop could work.

Reply:
We would need to support some kind of invalid state too, probably via an `isInvalid` prop that's rendered as an `aria-invalid` attribute.